### ISSUE TITLE
Access to builder

### ISSUE THREAD

**Issue by Erwane** [adamwathan/bootforms#87]
Hi,

There is a way to access to builder ? ( BootForm :: getBuilder() )
I need to build a select but only the select and your Form package is cool ;)

**Comment by adamwathan** [comment]
Do you actually want a plain unstyled select, or just not need the label? Could definitely add that method if needed!

**Comment by Erwane** [comment]
i need an unstyled select BUT binded to current form.

If i add this to BasicFormBuilder, i think it's ok :

<pre>	public function getBuilder()
	{
		return $this->builder;
	}
</pre>

**Comment by adamwathan** [comment]
Yeah sure, throw up a PR so you get the credit and I'll happily merge :+1:
